code/game/gamemodes/clock_cult/clock_scripture.dm

```diff
@@ -263,6 +267,7 @@ Judgement: 12 servants, 5 caches, 300 CV, and any existing AIs are converted or
 	var/ranged_type = /obj/effect/proc_holder/slab
 	var/ranged_message = "This is a huge goddamn bug, how'd you cast this?"
 	var/timeout_time = 0
+	var/allow_mobility = TRUE //if moving and swapping hands is allowed during the while
```

Comment:
Because all previous scriptures that used this allowed moving during the channel